ActindoWorkFlow/Overview/04_WorkflowProcessElements.md

```diff
@@ -2,29 +2,30 @@
 
 ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflow.png "[Workflow]")
 
-The *Workflows* module allows customers to model their own processes to match their companies structure. A workflow consists of a number of fixed elements that are chained together, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs.    
-For detailed information, see [Manage the workflows](../Operation/01_ManageWorkflows.md).
+The *Process Orchestration* module allows you to model your own processes to match your business processes. A workflow consists of a number of fixed elements that are chained together, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs.    
```

Comment:
Jetzt würde ich eher sagen: "... allows you to model your own workflows to match your business processes"